components/automate-gateway/api/event_feed/request/event.proto

```diff
@@ -4,9 +4,13 @@ package chef.automate.api.event_feed.request;
 option go_package = "github.com/chef/automate/components/automate-gateway/api/event_feed/request";
 
 message EventFilter {
+  // Filters to be applied to the request.
   repeated string filter = 1;
+  // Earliest events to return.
   int64 start = 2;
+  // Latest events to return.
   int64 end = 3;
+  // Count of events to return per page.
   int32 page_size = 4;
   int64 after = 5;
```

Comment:
i don't know what after, before, cursor, or collapse mean

Reply:
Ok, these fields are confusing because we are not using pagination. When requesting pages after the first one we need to know the last event's ID and date. The last event's ID goes in the "cursor" field and the last event's date goes in the "before" field.  For the event feed page, we only want events in descending order so we will only use the "before" field.  If an API user wanted to get pages in ascending order they would use the "after" field when requesting pages after the first one.

Reply:
The "collapse" field is if we want to group similar messages like below. 
![image](https://user-images.githubusercontent.com/1679247/85794498-97e29480-b6eb-11ea-9ee2-1db6ecbd0b0f.png)

Reply:
thank you!